python/get_python.sh

```diff
@@ -27,7 +27,17 @@ install_dependencies () {
         return $retVal
     fi
 
-    $DIR/pip.sh install -e $DIR/../scripts/o3de --no-deps --disable-pip-version-check  --no-warn-script-location
+    # if we're building a container app, create a package from o3de then install that to remove absolute paths to o3de scripts
+    if [ "$O3DE_PACKAGE_TYPE" == "SNAP" ]; then
+        pushd $DIR/../scripts/o3de/
+        $DIR/python.sh setup.py sdist
+        popd
+
+        $DIR/pip.sh install $DIR/../scripts/o3de/dist/o3de-1.0.0.tar.gz --no-deps --disable-pip-version-check --no-cache
+    else
+        $DIR/pip.sh install -e $DIR/../scripts/o3de --no-deps --disable-pip-version-check  --no-warn-script-location
+    fi
+
```

Comment:
this code is [already in development](https://github.com/o3de/o3de/blob/development/python/get_python.sh), strange that is showing up here?